cmd/sops-precommit/main.go

```diff
@@ -102,6 +106,14 @@ func main() {
 	}
 }
 
+func fileExists(filename string) bool {
```

Comment:
Should we start adding tests to this code base?

Reply:
Probably. I didn't have any tests because it was a POC but figured we would mature it if we liked the concept.